### PR TITLE
Fix vLLM prompt logprobs calculation

### PR DESCRIPTION
**Pull request by pan-x-c**
## Description

vLLM does not apply temperature scaling to logits when calculating prompt logprobs.
This PR adds the missing temperature scaling with a patch.

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: tests/common/vllm_test.py [hunk 1]
```diff
@@ -361,6 +363,87 @@ async def test_api(self):
...
+        self.assertTrue(response_1.logprobs is not None)
+        self.assertTrue(len(response_1.logprobs) > 0)
+        self.assertTrue(response_2.logprobs is not None)
+        self.assertTrue(len(response_2.logprobs) > 0)
+        logprobs_1 = self.model_wrapper.logprobs(response_1.tokens.tolist(), temperature=1.0)
+        logprobs_2 = self.model_wrapper.logprobs(response_1.tokens.tolist(), temperature=0.5)
+        logprobs_3 = self.model_wrapper.logprobs(response_2.tokens.tolist(), temperature=1.0)
+        logprobs_4 = self.model_wrapper.logprobs(response_2.tokens.tolist(), temperature=0.5)
+        self.assertEqual(logprobs_1.shape, logprobs_2.shape)
+        self.assertFalse(torch.equal(logprobs_1, logprobs_2))
```
Comment (hiyuchang) [inline tests/common/vllm_test.py]:
```suggestion
        self.assertEqual(logprobs_1.shape, logprobs_2.shape)
        self.assertEqual(logprobs_3.shape, logprobs_4.shape)
        self.assertFalse(torch.equal(logprobs_1, logprobs_2))
        self.assertFalse(torch.equal(logprobs_3, logprobs_4))
```

**Review by hiyuchang — APPROVED** [review]
